FBSnapshotTestCase/FBSnapshotTestController.h

```diff
@@ -41,6 +51,11 @@ extern NSString *const FBReferenceImageFilePathKey;
 @property (readwrite, nonatomic, assign) BOOL recordMode;
 
 /**
+ Save reference/captured/diffed images to disk after failures. Defaults to YES.
+ */
+@property (readwrite, nonatomic, assign) BOOL saveFailuresToDisk;
```

Comment:
What's the reason for introducing this property? In general I am not against but I would like to understand what's the use case.